sheetstorm/tests/test_unit_generate_test_data.py

```diff
@@ -327,3 +368,56 @@ def _compute_number_of_projects_to_create(options, projects_in_database, project
             raise UnsupportedProjectTypeException
 
         return number_of_projects_to_create
+
+
+class CreateDataFromPreparedSetTests(TestCase):
+    def test_get_request_to_create_data_using_prepared_set_function_should_return_True_if_any_set_requested(self):
+        options = {
+            DataSize.SMALL.name: True,
+            DataSize.MEDIUM.name: False,
+            DataSize.LARGE.name: False,
+            DataSize.EXTRA_LARGE.name: True,
+        }
+
+        self.assertTrue(GenerateTestDataCommand._get_request_to_create_data_using_prepared_set(options))
+
+    def test_get_request_to_create_data_using_prepared_set_function_should_return_True_if_not_any_set_requested(self):
```

Comment:
"...should_return_True_if_not_any_set_requested"? 🤔  It should be rephrased to something like "...should_return_false_if_paramter_is_not_provided"